src/schemas/CustomerObject.yaml

```diff
@@ -122,8 +122,3 @@ properties:
     type: array
     items:
       $ref: './CustomerMetadata.yaml'
```

Comment:
What about removing `metadata` as well? 🤔 

Seems they are not present by default when fetching an invoice:
https://github.com/getlago/lago-api/blob/main/app/serializers/v1/invoice_serializer.rb#L41

Reply:
They should: https://github.com/getlago/lago-api/pull/1172 (just fixed) :D